GitUI/UserControls/FileStatusList.cs

```diff
@@ -967,36 +950,36 @@ private void SelectItems(Func<ListViewItem, bool> predicate)
 
         private void SetDeleteFilterButtonVisibility()
         {
-            DeleteFilterButton.Visible = FilterVisibleInternal && !string.IsNullOrEmpty(FilterComboBox.Text);
+            DeleteFilterButton.Visible = FilterComboBox.Visible && !string.IsNullOrEmpty(FilterComboBox.Text);
         }
 
         private void SetFilterWatermarkLabelVisibility()
         {
-            FilterWatermarkLabel.Visible = FilterVisibleInternal && !FilterComboBox.Focused && string.IsNullOrEmpty(FilterComboBox.Text);
+            FilterWatermarkLabel.Visible = FilterComboBox.Visible && !FilterComboBox.Focused && string.IsNullOrEmpty(FilterComboBox.Text);
         }
 
         private void FileStatusListLoading()
         {
-            LoadingFiles.Visible = true;
+            // Show "Files loading" below the filterbox
             NoFiles.Visible = false;
+            int top = FilterComboBox.Visible ? FilterComboBox.Height + FilterComboBox.Margin.Bottom : 0;
```

Comment:
❔ in order to be prepared for future extensions:
```suggestion
            int top = FilterComboBox.Visible ? FilterComboBox.Bottom + FilterComboBox.Margin.Bottom : 0;
```

Reply:
Also in SetFileStatusListVisibility(), line 309